src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js

```diff
@@ -205,7 +205,6 @@ class EntityReferenceAutocomplete extends React.Component {
               fullWidth: true,
               label: this.props.label,
               InputProps: getInputProps({
-                disabled: this.state.loading,
```

Comment:
There is perhaps a bit of nuance here. If the request is inflight, do we want to send another request on every keystroke?

Reply:
Yeah, that makes a lot of sense. I'll take another stab at it and post another commit soon.

Reply:
>  If the request is inflight, do we want to send another request on every keystroke?

Why wouldn't we send another request on every keystroke?

Reply:
I think in general it's a bad idea to send a network request on each keystroke. That's potentially hundreds of requests for a single search. Perhaps adding a slight delay to attempt to guess when the user is done typing and issue the request then?

Reply:
Should we consider that if we run into any problems? I suggest that, for now, we do the request on every keystroke and improve it later if that is necessary.

Reply:
Drupal core ran with that strategy for a long time. I think that seems fine for itself